package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "version": "0.4.5",
+  "version": "0.4.8",
```

Comment:
why 0.4.5 > 0.4.8?

Reply:
agh sorry made edits from wrong branch

Reply:
current ver is 0.4.7 (with Coins constructor)

Reply:
got it